configure.ac

```diff
@@ -25,7 +25,7 @@ m4_define(libpcre2_posix_version, [3:5:0])
 AC_PREREQ([2.60])
 AC_INIT([PCRE2],pcre2_major.pcre2_minor[]pcre2_prerelease,[],[pcre2])
 AC_CONFIG_SRCDIR([src/pcre2.h.in])
-AM_INIT_AUTOMAKE([dist-bzip2 dist-zip])
+AM_INIT_AUTOMAKE([dist-bzip2 dist-zip foreign])
```

Comment:
This is not ideal... but I ended up adding "foreign" as the path of least resistance.

The problem is, I want to just add some "FOO.md" files to the tarball. That's totally utterly reasonable. And Automake 1.16 endorses this: it special-cases them, according to the fact the latest version of the GNU packaging standards say, "great, use FOO.md instead of FOO files".

But then you have the annoying issue, because Automake 1.15 does _NOT_ special-case them, it turned out to be really awkward to get the same behaviour from both Automake versions, just using the original configure.ac file.

So, this is the route I went down.

Autotools makes me cry (apologies if anyone is a fan...) but there's just so much time wasted every time you try and make modest changes, on every OSS or commercial project I've ever worked on that used Autotools!